types/multiavatar__multiavatar/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for @multiavatar/multiavatar 1.0
+// Project: https://multiavatar.com
```

Comment:
```suggestion
// Project: https://github.com/multiavatar/Multiavatar
```
just opinion, would be better for dev-exp I think, but you can skip this

Reply:
Agreed, it makes more sense to link to the GitHub repo in this context.
This was auto-generated by `dts-gen`.